CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ All notable changes to this project will be documented in this file.
 -   \[INTERNALS\] Stop storing `FunctionDef`, `ClassDef`, and `Import` objects inside `CodeBlock`s.
 -   \[INTERNALS\] Remove the `order` argument from the `pyccel.ast.core.Allocate` constructor.
 -   \[INTERNALS\] Remove `rank` and `order` arguments from `pyccel.ast.variable.Variable` constructor.
+-   \[INTERNALS\] All internal classes which can be generated from `FunctionCall`s must inherit from `TypedAstNode`.
```

Comment:
In the PR description you mention that all built-in classes must inherit from `PyccelInternalFunction`, not just `TypedAstNode`.

Reply:
Fixed f8163f2b3

Reply:
Just wondering. Do we still have a reason to call it `PyccelInternalFunction` rather than just `PyccelFunction`?

Reply:
No I don't think so

Reply:
How about making a search & replace in this PR?

Reply:
Done 3c478b38a
and CHANGELOG 7c67d46

Reply:
Great! The PR description also needs updating.

Reply:
Done